app/models/goldencobra/template.rb

```diff
@@ -0,0 +1,22 @@
+module Goldencobra
+  class Template < ActiveRecord::Base
+    attr_accessible :title, :layout_file_name
```

Comment:
Das ein Template ein Layout-File hat, find ich passt von der Namensgebung.
Template ist etwas übergeordnetes, was in Zukunft ja auch noch mehr haben kann.
Und in Rails sind es ja eindeutig Layout-Files.